_data/events.yml

```diff
@@ -23,19 +23,47 @@
   # R or Python
   program_language: Any
   # Only one label that will be put into the GitHub Issue
-  gh_labels: lightning-demos
-  
-- key: lightning-demos-python
+  gh_labels: code-along
+
+  # Key is the part of the filename in _posts/
+- key: intro-r
+  # Title of event that will show up as the GH Issue title
+  title: "Introduction to R"
+  # Explanation of event as a paragraph form
+  description: >
+    A brief introduction to R for beginners.
+  # Date of the event in YYYY-MM-DD format
+  date: 2021-01-20
+  # Start and end time for event in 24 hour HH:MM format
+  start_time: "18:00"
+  end_time: "19:00"
+  # Where the event will take place
+  location: Zoom
+  # The URL for the location
+  location_url: ""
+  # Packages to install, a space separating each package
+  # (e.g. "dplyr tidyr" or "Pandas>=0.21 Seaborn")
+  packages: ""
+  youtube_link: ""
+  skill_level: Beginner
+  # Beginner Intermediate Advanced
+  # R or Python
+  program_language: Any
```

Comment:
And this one R?

Reply:
Good catch, this is what I get for leaving putting this together for the last task of a long day... on it